codespell_lib/data/dictionary.txt

```diff
@@ -23308,6 +23316,7 @@ spoonfulls->spoonfuls
 sporatic->sporadic
 sporious->spurious
 sppeches->speeches
+spped => speed, skipped, swapped, stripped,
```

Comment:
Need to look at the [error log](https://travis-ci.org/github/codespell-project/codespell/jobs/681582401#L537-L540):
```
>               err, rep = line.decode('utf-8').split('->')
538E               ValueError: not enough values to unpack (expected 2, got 1)
539
540err        = 'sppeches'
```
This suggests that something in the vicinity of `sppeches` does not contain a `->`, looks like it's this one

Reply:
Looks better now. Thank you!